### PR TITLE
Misc. drivetrain improvements

### PR DESCRIPTION
**Pull request by simon-andrews**

### CODE REVIEW

File: src/org/robockets/steamworks/Robot.java [hunk 1]
```diff
@@ -153,9 +153,21 @@ public void robotInit() {
 		RobotMap.rollerEncoderCounter.setDistancePerPulse(1.0);
 		
 		RobotMap.shooterRollerSpeedController.setInverted(true);
+		RobotMap.climberSpeedController.setInverted(true);
+		RobotMap.climberSpeedController2.setInverted(true); // 55left 75 right
 
-		RobotMap.leftEncoder.setDistancePerPulse(0.26592797783933518005540166204986);
-		RobotMap.rightEncoder.setDistancePerPulse(0.04164859002169197396963123644252);
+		RobotMap.leftEncoder.setDistancePerPulse(0.03435326842837273991655076495132);
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/Robot.java]: Could be a constant with a value of`4 * Math.PI / 360`. 0.34... is a magic number.

File: src/org/robockets/steamworks/Robot.java [hunk 2]
```diff
@@ -167,7 +179,7 @@ private void initSmartDashboard() {
 		// CLIMBER //
 		/////////////
 		Robot.climber.initSmartDashboard();
-		SmartDashboard.putData("Climb", new Climb(1));
+		SmartDashboard.putData("Climb", new Climb(-1));
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/Robot.java]: It shouldn't be possible to change the direction of the climber with a parameter. The climber can only go one direction, and that direction should be a constant that is tucked away where no one else can modify it.

TBH, the `Victor` objects for the climber motors shouldn't even be available in `RobotMap.`

File: src/org/robockets/steamworks/RobotMap.java [hunk 3]
```diff
@@ -132,7 +134,8 @@
     /**
      * PDP port that the climber's motor is attached to.
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/RobotMap.java:135]: JavaDoc needs updating.

File: src/org/robockets/steamworks/climber/Climber.java [hunk 4]
```diff
@@ -14,7 +14,12 @@
 	 * @return current The current of the motor.
 	 */
 	public double readCurrent() {
-		return RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPort);
+		SmartDashboard.putNumber("current LEFT", RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortLeft));
+		SmartDashboard.putNumber("current RIGHT", RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortRight));
+		
+		return (
+				Math.abs(RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortLeft)) +
+				Math.abs(RobotMap.powerDistPanel.getCurrent(RobotMap.climberPdpPortRight))) / 2.0;
```
Comment (simon-andrews) [inline src/org/robockets/steamworks/climber/Climber.java:22]: This is really gross. There should be a variable for the current values that is both written to SmartDashboard and used in the average calculation.

And yes, for the record, I did write this. I'm cut, I know.

**Review by simon-andrews — COMMENTED** [review]
I opened this pull request so GitHub says I can't request changes, but I'm requesting changes anyway.

File: src/org/robockets/steamworks/climber/Climber.java (hunk 4 above)
Comment (UnknownSilicon) [reply src/org/robockets/steamworks/climber/Climber.java:22]: Scissors

File: src/org/robockets/steamworks/autonomous/EasyAuto.java [hunk 5]
```diff
@@ -32,18 +31,18 @@ public EasyAuto(int startingPos) {
 			forwardDistance1 = 60.785;
 			forwardDistance2 = 74.575;
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/autonomous/EasyAuto.java:32]: These may need to be changed again

**Comment by simon-andrews** [comment]
:+1:

**Review by UnknownSilicon — APPROVED** [review]
:+1:

**Pull request merged**